pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1483,6 +1518,10 @@ func TestTryRegisterWithApiServer(t *testing.T) {
 	}
 
 	for _, tc := range cases {
+
+		if tc.getOnForbiddenDisabled {
+			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DisableKubeletRegisterGetOnForbidden, true)
```

Comment:
```suggestion
			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DisableKubeletRegisterGetOnForbidden, true)()
```

Reply:
Agree on the scope of the override - it would apply to the top level test, but the cleanup is done by a t.Cleanup() at https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/featuregate/testing/feature_gate.go#L71

So the current usage is correct.